src/bezpath.rs

```diff
@@ -942,6 +943,35 @@ impl PathSeg {
             PathSeg::Cubic(cubic_bez) => cubic_bez.is_nan(),
         }
     }
+
+    #[inline]
+    fn as_vec2_vec(&self) -> Vec<Vec2> {
+        match self {
+            PathSeg::Line(l) => vec![l.p0.to_vec2(), l.p1.to_vec2()],
+            PathSeg::Quad(q) => vec![q.p0.to_vec2(), q.p1.to_vec2(), q.p2.to_vec2()],
+            PathSeg::Cubic(c) => {
+                vec![
+                    c.p0.to_vec2(),
+                    c.p1.to_vec2(),
+                    c.p2.to_vec2(),
+                    c.p3.to_vec2(),
+                ]
+            }
+        }
+    }
+
+    /// Minimum distance between two PathSegs
```

Comment:
Some questions. Do the pathsegs need to be of the same type (line/quad/cubic), or can it do the degree raising as needed?

Also, I see it returning the distance, t1, and t2. This should be stated in the docstring.

Reply:
The segs don't need to be the same order, but it doesn't do degree raising either - it works in terms of basis functions, and it has sums that do things like `∑i=0..order(Curve1) ∑j=0..order(Curve2) f(Curve1, i) ⋅ g(Curve2, j)`.